static/coffee/locations.coffee

```diff
@@ -270,16 +291,20 @@ initMap = (id, geojson, question, districtLabel) ->
     html = ""
 
     scaleClass = 'national'
-    if level
+    if level and level == 1
       scaleClass = 'state'
+    if level and level == 2
+      scaleClass = 'district'
 
     html = "<div class='scale " + scaleClass + "'>"
     html += "<div class='scale-map-circle-outer primary-border-color'></div>"
     html += "<div class='scale-map-circle-inner'></div>"
     html += "<div class='scale-map-hline primary-border-color'></div>"
     html += "<div class='scale-map-vline primary-border-color'></div>"
+    html += "<div class='scale-map-vline-2 primary-border-color'></div>"
```

Comment:
here you could probably use [block strings and string interpolation](http://coffeescript.org/#strings)